module-1/lab-numpy/your-code/main.py

```diff
@@ -0,0 +1,197 @@
+#1. Import the NUMPY package under the name np.
+
+import numpy as np
+
+#2. Print the NUMPY version and the configuration.
+
+print (np.__version__)
+
+#3. Generate a 2x3x5 3-dimensional array with random values. Assign the array to variable "a"
+# Challenge: there are at least three easy ways that use numpy to generate random arrays. How many ways can you find?
+
+a = np.random.random((2,3,5))
+print (a)
+
+n=np.random.randint(1000000)
+print(n)
+a = np.random.randint(n, size=(2,3,5))
+print (a)
+
+
+n = np.random.random (100000)
+a=np.random.choice(n, size= (2,3,5))
+print (a)
+
+#4. Print a.
```

Comment:
You don't need to write again the assignation of a. It's just print (a), as you have defined this variable above.